### ISSUE TITLE
MONDO:0002010 FG Syndrome

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#2544]
MONDO:0002010 , "FG Syndrome" (with multiple "FGS" subtype children.)
Current Xrefs:

UMLS:C0220769   \| OMIMPS:305400 \| ORPHANET: 323
--
The current UMLS xref is to FGS type 1 and there is no link to the OMIM PS from number OMIM 305400, possibly because the disorder ("FGS 1") is now called OPITZ-KAVEGGIA SYNDROME; OKS. The MONDO term itself seems to be a bit of an "orphan" with poorly define hierarchy above.  The OrphaNet ID 323 returns no hits on orpha.net. So which are the appropriate children? OMIM doesn't currently link 305400 to this phenotypic series, though it is a live page as far as I can tell: http://www.omim.org/phenotypicSeries/PS305400
For FGS parent concept: The UMLS X-ref seems wrong, but UMLS seems to have no parent term for the various subtypes of FGS (all the FGS terms are indeed on the FGS1 term cited above.)

Can we figure out what to do with this term? Does OKS (OMIM305400) belong as a child of FGS types? 
Based on discussion here, I will open a ticket with UMLS to get an appropriate parent term made, as needed. 
@nicolevasilevsky can you please tag the appropriate individuals from OMIM and OrphaNet to weigh in here?

**Comment by nicolevasilevsky** [comment]
Thanks @kanems - I'll look into this asap

**Comment by nicolevasilevsky** [comment]
> Does OKS (OMIM305400) belong as a child of FGS types?

Yes, it should be a subclass because it is part of the phenotypic series. I'll make this change.

**Comment by nicolevasilevsky** [comment]
@ana-rath-orphanet @ahamosh 
Please see the questions above - thank you!

**Comment by sabrinatoro** [comment]
Hello @kanems 
I'm Sabrina, a new curator who recently joined the Mondo team.
We will address your issue as soon as possible. Thank you for your patience.
If this is a high priority, please let me know and we can prioritize it. Thank you!

**Comment by nicolevasilevsky** [comment]
to do:
- [ ] UMLS:C0220769 should be MONDO:superClassOf on 'FG syndrome'
- [ ] UMLS:C0220769 should be MONDO:equivalentTo on 'FG syndrome 1'
- [ ] Orphanet:323 should be MONDO:equivalentObsolete on 'FG syndrome'

@kanems 
OMIM:305400 seems to be an appropriate child of 'FG syndrome'
![image](https://user-images.githubusercontent.com/6722114/136250566-cf7366bf-253c-450d-b116-1973bbdcd072.png)

**Comment by kanems** [comment]
@nicolevasilevsky Thank you for following up on this. I will review and see if I need to add this to my list of questions for UMLS.
